src/App.tsx

```diff
@@ -26,57 +27,103 @@ type ReorderOptions = {
   isReversed: boolean;
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
-  const visibleGoods = [...goods];
+  let visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  switch (sortType) {
+    case SortType.NONE:
```

Comment:
Probably, you can combine this case with default. 

Not sure that you want to just break on a default case. I guess we may assume that default === no sorting here. 
However, it's up to you.